Doppler.Sap/Enums/ResponsabileBillingEnum.cs

```diff
@@ -0,0 +1,26 @@
+namespace Doppler.Sap.Enums
+{
+    public enum ResponsabileBillingEnum
+    {
+        GB = 1,
+
+        QBL = 2,
+
+        QuickBookUSA = 3,
+
+        QuickBookLATAM = 4,
+
+        RC = 5,
+
+        VVD = 6,
+
+        RafaelSanhueza = 7,
```

Comment:
Did you check if all of these billing systems are updated? I'm not sure if this one is in use.

Reply:
Nop, I created the enum similiar to Doppler. I will talk about it with Noe.

Reply:
Also, we should have into account that the billing systems are often replaced or modified.  Maybe notify the billing team of this, so that any changes in doppler go in hand with this service. 
I'm not sure if it will be needed though...

Reply:
Updated about Noe's comment. Now the updated enum is:
```
public enum ResponsabileBillingEnum
    {
        QBL = 2,
        QuickBookUSA = 3,
        RC = 5,
        GBBISIDE = 9,
        Mercadopago = 13,
        BorisMarketing = 16
    }
```